content/300-guides/050-database/100-developing-with-prisma-migrate/160-baselining.mdx

```diff
@@ -23,6 +16,13 @@ Baselining tells Prisma Migrate to assume that one or more migrations have **alr
 
 Baselining is part of [adding Prisma Migrate to a project with an existing database](add-prisma-migrate-to-a-project).
 
+<Admonition type="info">
+
+This guide **does not apply for MongoDB**.<br />
+Instead of `migrate deploy`, [`db push`](/concepts/components/prisma-migrate/db-push) is used for [MongoDB](/concepts/database-connectors/mongodb).
+
+</Admonition>
```

Comment:
Point out that Migrate for MongoDB is `db push` instead of "not supported" warning.